openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -132,35 +135,56 @@ All OAuth 2.0 Grant Types and extensions mechanisms can be used in conjunction w
-* A new authorization details [@!RFC9396] type `openid_credential` is defined to convey the details about the Credentials (including formats and types) the Wallet wants to obtain (see (#authorization-details)).
-* The Token Response error codes `authorization_pending` and `slow_down` are used to allow for deferred authorization of Credential issuance. These error codes are supported for the Pre-Authorized Code grant type.
+* A new authorization details [@!RFC9396] type `openid_credential` is defined to convey the details about the Credentials (including Credential Dataset, formats, and types) the Wallet wants to obtain (see (#authorization-details)).
+* New token response error codes `authorization_pending` and `slow_down` are added to allow for deferred authorization of Credential issuance. These error codes are supported for the Pre-Authorized Code grant type.
 * Client metadata is used to convey the Wallet's metadata. The new Client metadata parameter `credential_offer_endpoint` is added to allow a Wallet (acting as OAuth 2.0 client) to publish its Credential Offer Endpoint (see (#client-metadata)).
 * Authorization Endpoint: The additional parameter `issuer_state` is added to convey state in the context of processing an issuer-initiated Credential Offer (see (#credential-authz-request)). Additional parameters `wallet_issuer` and `user_hint` are added to enable the Credential Issuer to request Verifiable Presentations from the calling Wallet during Authorization Request processing.
 * Token Endpoint: Optional response parameters `c_nonce` and `c_nonce_expires_in` are added to the Token Endpoint, Credential Endpoint, and Batch Credential Endpoint to provide the Client with a nonce to be used for proof of possession of key material in a subsequent request to the Credential Endpoint (see (#token-response)).
 
 ## Core Concepts
 
-The Wallet sends one Credential Request per individual Credential to the Credential Endpoint. 
+In the context of this specification, Credential Datasets define the data (claims) about a subject that is to be included in a Credential.
+The Credential Format defines how the data (or Dataset) within final Verifiable Credential is organized and secured.
+This can include the specific parameters needed to describe the Credential, which are established in the Credential Format Profile.
+While in principle independent of each other, the Credential Dataset and the Credential Format can have a relationship in the sense that an Issuer may only offer certain Credential Formats for certain Credential Datasets.
 
-To request issuance of multiple Credentials of the same or different types/doctypes bound to the same or different proofs the Wallet MAY:
+An End-User typically authorizes the issuance of Credentials with a specific Credential Dataset, but do not usually care about the Credential Format. The same Credential Dataset may even be issued in different Credential Formats.
```

Comment:
The spec until now doesn't really make statements about how/where user consent takes place. In the most prominent setup today, Wallets ask for consent after the Credential Offer and there is only information about Credential Configuration but not the Dataset